tst/EnergyPlus/unit/SimAirServingZones.unit.cc

```diff
@@ -1568,4 +1569,530 @@ TEST_F(EnergyPlusFixture, InitAirLoops_2AirLoop3ADUb)
     EXPECT_EQ(DataZoneEquipment::ZoneEquipConfig(2).InletNodeAirLoopNum(2), 2);
 }
 
+TEST_F(EnergyPlusFixture, AirLoop_ReturnFan_MinFlow)
+{
+
+    // Test for #6050 - Return Fan 'Fan Power Minimum Flow Fraction' enforces loop flow, while it should not
+    // I am simulating a very simple shoebox building with 1 zone.
+    // There is an airLoopHVAC with a Return Fan:VariableVolume, with a 'Fan Power Minimum Flow Fraction' = 0.3
+    // The Supply fan has a fraction of zero
+    //
+    // The supply side of the loop is has follows: only has a Return Fan, a Mixing box, and a SupplyFan
+    //                        o   o
+    //                        |   |
+    //  --- (Return Fan) ---- o-/-o ---- (SupplyFan) ----
+    //  |                                               |
+    //  o                                               o
+    //
+    //  The Demand side has only one ATU:VAV:NoReheat with a Minimum flow Fraction of 0 so that it doesn't drive the airlfow either
```

Comment:
Gist of the unit test./ This is basically exactly the MCVE I created on EnergyPlusDevSupport.